src/hotspot/cpu/aarch64/macroAssembler_aarch64.hpp

```diff
@@ -1113,10 +1113,51 @@ class MacroAssembler: public Assembler {
 #endif
 
 public:
-  // Calls
+  // AArch64 OpenJDK uses four different types of calls:
+  //   - direct call: bl pc_relative_offset
+  //     This is the shortest and the fastest, but the offset has the range:
+  //     +/-128MB for the release build, +/-2MB for the debug build.
+  //
+  //   - far call: adrp reg, pc_relative_offset; add; bl reg
+  //     This is longer and slower than a direct call. The offset has
```

Comment:
```suggestion
  //     This is longer than a direct call. The offset has
```
I wouldn't be so definite as to say it's slower. Let's stay with longer.